Makefile

```diff
@@ -30,7 +31,7 @@ install: $(TARGET) $(DOCDIR)/$(MAN1)
 uninstall:
 	$(RM) $(DESTDIR)$(BINDIR)/$(TARGET)
 	$(RM) $(DESTDIR)$(MANDIR)/man1/$(MAN1)
-	$(RM) -r $(DESTDIR)$(EXAMPLEDIR)
+	$(RM) -r $(SHAREDIR)
```

Comment:
Yes, you're right. How does it work, do I change this, cancel this PR and create another one, or is there a cleaner solution ?